extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -647,18 +648,26 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
                RTC_ATI_ENUM.RTC_FAILURE);
       ard.push(rtcResponse.callout);
       if (rtcResponse.response) {
-        ['targeting', 'categoryExclusions'].forEach(key => {
-          if (rtcResponse.response[key]) {
-            const rewrittenResponse = this.rewriteRtcKeys_(
-                rtcResponse.response[key],
-                rtcResponse.callout);
-            this.jsonTargeting_[key] =
-                !!this.jsonTargeting_[key] ?
-                deepMerge(this.jsonTargeting_[key],
-                    rewrittenResponse) :
-                rewrittenResponse;
-          }
-        });
+        if (rtcResponse.response['targeting']) {
+          const rewrittenResponse = this.rewriteRtcKeys_(
+              rtcResponse.response['targeting'],
+              rtcResponse.callout);
+          this.jsonTargeting_['targeting'] =
+              !!this.jsonTargeting_['targeting'] ?
+              deepMerge(this.jsonTargeting_['targeting'],
+                  rewrittenResponse) :
+              rewrittenResponse;
+        }
+        if (rtcResponse.response['categoryExclusions']) {
+          [this.jsonTargeting_['categoryExclusions'],
```

Comment:
Only need to initialize exclusions once so above do:
let exclusions;
...
if (rtcResponse.response['categoryExclusions']) {
   if (!exclusions) {
      exclusions = {};
      // populate it here
   }
   // iterate response cats
}
// Outside of rtc response array iteration
if (exclusions) {
    this.jsonTargeting_['cat...'] = Object.keys(exclusions);
}